send_comments.py

```
@@ -0,0 +1,82 @@
+import requests
+import os
+from dotenv import load_dotenv
+from app.utils.logger import logger
+
+load_dotenv()
+github_token = os.getenv("GITHUB_TOKEN_KEY")
```

Environment variable name 'GITHUB_TOKEN_KEY' is unconventional. Ensure the variable is set and consider using 'GITHUB_TOKEN'. Also validate token presence and fail early with a clear error if missing.